homeassistant/components/systemmonitor/sensor.py

```diff
@@ -482,6 +500,36 @@ def get_arguments() -> dict[str, Any]:
             )
             continue
 
+        if _type == "process_num_fds":
+            # Create sensors for processes configured in binary_sensor section
+            processes = entry.options.get(BINARY_SENSOR_DOMAIN, {}).get("process", [])
```

Comment:
This feels a bit wrong, but that is how we can reuse the configuration from the existing process sensor. Previously this used to be a sensor and got converted to a binary_sensor with #108585.

Do we have a common pattern if we use such configuration for multiple platforms?